docs/content/reference/install-configuration/entrypoints.md

```diff
@@ -0,0 +1,238 @@
+---
+
+Listening for Incoming Connections/Requests
+{: .subtitle }
+
+### Configuration Example
+
+```yaml tab="File (YAML)"
+entryPoints:
+  web:
+    address: :80
+    http:
+      redirections:
+        entryPoint:
+          to: websecure
+          scheme: https
+          permanent: true
+
+  websecure:
+    address: :443
+    tls: {}
+    middlewares:
+      - auth@kubernetescrd
+      - strip@kubernetescrd
+```
+
+```yaml tab="Helm Chart Values"
+## Values file
+ports:
+  web:
+    port: :80
+  websecure:
+    port: :443
+    tls:
+      enabled: true
+    middlewares:
+      - auth@kubernetescrd
+      - strip@kubernetescrd
+additionalArguments:
+  - --entryPoints.web.http.redirections.to=websecure
+  - --entryPoints.web.http.redirections.scheme=https
+  - --entryPoints.web.http.redirections.permanent=true
+```
+
+!!! tip 
+
+      In the Helm Chart, the `entryPoints` `web` (port 80), `websecure` (port 443), `traefik` (port 9000) and `metrics` (port 9100) are created by default.
+      The `entryPoints` `web`, `websecure` are exposed by default using a Service.
+
+      The default behaviors can be overridden in the Helm Chart.
+
+## Configuration Options
+
+<!-- markdownlint-disable MD013 -->
+
+| Field | Description | Default | Required |
+|:------|:------------|:--------|:---------|
+| `address` | Define the port, and optionally the hostname, on which to listen for incoming connections and packets.<br /> It also defines the protocol to use (TCP or UDP).<br /> If no protocol is specified, the default is TCP. The format is:`[host]:port[/tcp\|/udp]`.<br /> More information [here](#address). |  | Yes |
+| `asDefault` | Mark the `entryPoint` to be in the list of default `entryPoints`.<br /> `entryPoints`in this list are used (by default) on HTTP and TCP routers that do not define their own `entryPoints`option.<br /> More information [here](#asdefault).  | false | No |
+| `forwardedHeaders.trustedIPs` | Set the IP from where Traefik trusts the forwarded headers information (`X-Forwarded-*`). | | No |
+| `forwardedHeaders.insecure` | Set the insecure mode to always trust the forwarded headers information (`X-Forwarded-*`).<br />We recommend to use this option only for tests purposes, not in production. | false | No |
+| `http.redirections.`<br />`entryPoint.to` | The target element to enable (permanent) redirecting of all incoming requests on an entry point to another one. <br /> The target element, it can be an entry point name (ex: `websecure`), or a port (`:443`). | - | Yes |
+| `http.redirections.`<br />`entryPoint.scheme` | The target scheme to enable (permanent) redirecting of all incoming requests on an entry point to another one changing the scheme. <br /> The target element, it can be an entry point name (ex: `websecure`), or a port (`:443`). | "https" | No |
```

Comment:
```suggestion
| `http.redirections.`<br />`entryPoint.scheme` | The target scheme to enable (permanent) redirecting of all incoming requests on an entry point to another one changing the scheme. <br /> The target element, it can be an entry point name (ex: `websecure`), or a port (`:443`). | https | No |
```